addons/survey/wizard/survey_invite.py

```diff
@@ -61,6 +63,22 @@ def _get_default_author(self):
     deadline = fields.Datetime(string="Answer deadline")
     send_email = fields.Boolean(compute="_compute_send_email",
                                 inverse="_inverse_send_email")
+    survey_type = fields.Selection(related="survey_id.survey_type")
+    scheduled_date = fields.Char(
+        'Scheduled Date',
+        compute="_compute_schedule_date", readonly=False, store=True,
+        help="send emails after that date. This date is considered as being in UTC timezone."
+    )
+    model = fields.Char('Related Document Model', compute='_compute_model', readonly=False, store=True)
```

Comment:
Useless, to remove (and badly named anyway). We don't need to this the same value, what matters is the render model.